.github/workflows/calypso-live.yml

```diff
@@ -26,5 +27,5 @@ jobs:
               issue_number: context.issue.number,
               owner: context.repo.owner,
               repo: context.repo.repo,
-              body: '<!--calypso-live-watermark:apr@v1-->\nLink to Calypso live: ${{ steps.build_link.outputs.LINK }}\nLink to Jetpack Cloud live: ${{ steps.build_link.outputs.LINK }}&env=jetpack\n_(These links may take a few minutes to be available)_'
+              body: '<!--calypso-live-watermark:apr@v1--><details> <summary>Calypso Live <a href="${{steps.build_link.outputs.LINK}}">(direct link)</a></summary> <table> <tr> <td> <img src="${{steps.build_link.outputs.QR_CALYPSO}}"/> </td><td> <a href="${{steps.build_link.outputs.LINK}}">${{steps.build_link.outputs.LINK}}</a> </td></tr></table> </details> <details> <summary>Jetpack Cloud live <a href="${{steps.build_link.outputs.LINK}}&env=jetpack">(direct link)</a></summary> <table> <tr> <td> <img src="${{steps.build_link.outputs.QR_JETPACK}}"/> </td><td> <a href="${{steps.build_link.outputs.LINK}}&env=jetpack">${{steps.build_link.outputs.LINK}}&env=jetpack</a> </td></tr></table> </details>'
```

Comment:
Could we keep the "These links may take a few minutes to be available" message? I think it's still useful.

Reply:
Yes In my testing the links never really worked. so I think it would be good to have them. I think they could be useful in some cases but I am not sure exactly when. Let's remove them in a follow-up PR.